### PR TITLE
Changes to accomodate additional loss.

### PR DESCRIPTION
**Pull request by hy23-lfb**

### CODE REVIEW

File: scripts/tf/train.py [hunk 1]
```diff
@@ -183,17 +184,19 @@ def __init__(self, n_gradients, *args, **kwargs):
...
+        print("############### inside train_step ###################")
         self.n_acum_step.assign_add(1)
 
         x, y = data
+
         # Gradient Tape
         with tf.GradientTape() as tape:
             y_pred = self(x, training=True)
             loss = self.compiled_loss(y, y_pred, regularization_losses=self.losses)
+            print("type of loss : {}".format(type(loss)))
```
Comment (hy-23) [inline scripts/tf/train.py:198]: remove these

**Review by hy-23 — CHANGES_REQUESTED** [review]

**Review by hy-23 — APPROVED** [review]
